Keas.Mvc/ClientApp/components/Documents/DocumentsContainer.tsx

```diff
@@ -82,37 +83,37 @@ export const DocumentsContainer = (props: IProps): JSX.Element => {
   );
 
   if (!canView) {
-    return <Denied viewName='Documents' />;
+    return <Denied viewName="Documents" />;
   }
 
   if (loading) {
     return <div>Loading Documents...</div>;
   }
 
   return (
-    <div className='card documents-color'>
-      <div className='card-header-documents'>
-        <div className='card-head row justify-content-between'>
+    <div className="card documents-color">
+      <div className="card-header-documents">
+        <div className="card-head row justify-content-between">
           <h2>
-            <i className='fas fa-file-alt fa-xs' /> Documents
+            <i className="fas fa-file-alt fa-xs" /> Documents
           </h2>
-          <Button color='link' onClick={() => setAssign(true)}>
-            <i className='fas fa-plus fa-sm' aria-hidden='true' /> Add Document
+          <Button color="link" onClick={() => setAssign(true)}>
+            <i className="fas fa-plus fa-sm" aria-hidden="true" /> Add Document
           </Button>
         </div>
       </div>
-      <div className='card-content'>
+      <div className="card-content">
         <DocumentsList
           documents={documents}
           downloadUrl={`/api/${ctx.team.slug}/documents/get`}
-        ></DocumentsList>
```

Comment:
Prettier?

Reply:
Yea but I wonder why my prettier is slightly different. I thought they would all use the ruleset that is in the repo.